docs/faq/questions/cloud-faq.mdx

```diff
@@ -233,3 +233,9 @@ Cloud.
 - `https://docs.cypress.io` - **Cypress documentation**
 - `https://download.cypress.io` - **CDN download of Cypress binary**
 - `https://on.cypress.io` - **URL shortener for link redirects**
+
```

Comment:
I'm re-reading this whole section for better context and think we can make a couple or edits.

First, the section title reads if you work _at_ a VPN, which likely should be changed to _with_ unless theres something here I'm not aware of. 

Also, the section about the Dedicated IPs... we say you might need to add these two. Can you provide more context here? Is that the same as allowing URLs like above? I might read this and say "add these both where?"